_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,159 @@
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a new proposed Bitcoin P2P protocol
+message, a BIP for the bech32 modified address format, and an idea for
+preventing UTXO probing in proposed dual-funded LN channels.  Also
+included are our regular sections with the summary of a Bitcoin Core PR
+Review Club meeting, a list of releases and release candidates, and
+descriptions of notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Proposed `disabletx` message:** in 2012, [BIP37][] was published,
+  giving lightweight clients the ability to request that peers not
+  relay to the client any unconfirmed transactions until the client
+  had loaded a [bloom filter][topic transaction bloom filtering].
+  Bitcoin Core [later][bitcoin core #6993] reused this mechanism for its
+  bandwidth-saving `-blocksonly` mode where a node requests that its
+  peers don't send it any unconfirmed transactions at all.  Last year,
+  Bitcoin Core with default settings began opening a few
+  block-relay-only connections as an efficient way to improve [eclipse
+  attack][topic eclipse attacks] resistance without significantly
+  increasing bandwidth or reducing privacy (see [Newsletter #63][news63
+  bcc15759]).  However, the BIP37 mechanism being used to suppress
+  transaction relay allows the initiating node to request full
+  transaction relay at any time.  Transaction relay can consume hundreds
+  of megabytes of data per day per peer, so nodes need to set their anti
+  Denial of Service (DoS) limits with the assumption that each
+  BIP37-based low-bandwidth blocks-relay-only connection could suddenly
+  become a full transaction relay connection.
+
+    This week, Suhas Daftuar [posted][daftuar disabletx] to the
+    Bitcoin-Dev mailing list a new proposed BIP for a `disabletx`
+    message that could be sent during connection negotiation.  A peer
+    that understands the message won't send the node requesting
+    `disabletx` any transaction announcements and won't request any
+    transactions from the node.  It also won't send some other gossip
+    messages such as `addr` messages used for IP address announcements.
+    The `disabletx` negotiation persists for the lifetime of the
+    connection, allowing peers to exclude disabled relay connections
+    from their transaction relay anti-DoS rules.
+
+- **Bech32m** Pieter Wuille [posted][wuille bech32m post] to the
+  Bitcoin-Dev mailing list a [draft BIP][bech32m bip] for a modified
+  version of the [bech32][topic bech32] algorithm that increases the
+  chance that the accidental addition or removal of characters in one of
+  these *bech32m* addresses will be detected.  If no problems are found
+  with the proposal, it is expected that bech32m addresses will be used
+  for [taproot][topic taproot] addresses and possibly future new script
+  upgrades.
+
+- **LN dual funding anti UTXO probing:** a long-term goal for LN has
```

Comment:
Did you mean to link to Lloyd's post: https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-January/002929.html ?